### PR TITLE
Add german translation

### PR DESCRIPTION
**Pull request by midnightrun**
Hi,
this PR adds a german translation to Send2KindleBot. It is written in the polite way of talking in german 

Cheers 🍺!

### CODE REVIEW

File: i18n/bot.de-de.yml [hunk 1]
```diff
@@ -0,0 +1,87 @@
+en-us:
```
Comment (GabrielRF) [inline i18n/bot.de-de.yml]: Could you change this to `de:`?

Comment (midnightrun) [reply i18n/bot.de-de.yml]: Hi Gabriel,
I changed it to de-de because maybe you will get some other german idioms like Austria german (de_AT).

Chéers!

Comment (GabrielRF) [reply i18n/bot.de-de.yml]: I've tested here, on my mobile device. 
When I set Deutsch as the language, Telegram says only `de` to the bot.

Comment (midnightrun) [reply i18n/bot.de-de.yml]: Sorry, didn't know that. Fixed :)

**Comment by midnightrun** [comment]
Sorry, I don't user Telegram myself :)

**Comment by GabrielRF** [comment]
@midnightrun Don't worry. I guess I've fixed already.

**Pull request merged**